themes/aframe/source/css/_settings.styl

```diff
@@ -47,6 +47,12 @@ media--mobile-device() {
   }
 }
 
+media--mobile-only() {
```

Comment:
I agree that min-width is preferable, though I wrote it with a max-width query because I'm overriding some of the auto complete default styles. If I used a min-width I'd have to override them for mobile, and then duplicate the originals for larger view ports.

Would it be okay if I remove the `+media--mobile-only()` helper and write the max-width media query directly inline to it looks more like a one-off?